frontend/src/minuta/ListaActionItems.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import Card from '@material-ui/core/Card';
+import Divider from '@material-ui/core/Divider';
+import {ListaActionItemsContainer,
+        ActionItemDesciption,
+        ActionItemContainer,
+        Owner} from './ListaActionItems.styled'
+
+const actionItem = (descripcion, owners) =>{
+    return (
+        <ListItem>
+            <ActionItemContainer>
+            <ActionItemDesciption>{descripcion}</ActionItemDesciption>
+            <div>
+              { owners.map((owner) => <Owner>{"@" + owner}</Owner> ) }
+            </div>
+            </ActionItemContainer>
+        </ListItem>
+    )
+}
```

Comment:
Debería ser un componente...

```suggestion
const ActionItem = ({ descripcion, owners }) => (
    <ListItem>
        <ActionItemContainer>
            <ActionItemDesciption>{descripcion}</ActionItemDesciption>
            <div>
              { owners.map((owner) => <Owner>{"@" + owner}</Owner> ) }
            </div>
        </ActionItemContainer>
    </ListItem>
)
```

Reply:
si te digo que esta arreglado en otro pr sirve? Para evitar confiltos de merge preferiria dejar que el otro pr lo resuelva

Reply:
el otro pr tiene todo lo de este + otra funcionalidad mas, quiza conviene mergiar el otro de una?